package.json

```diff
@@ -25,6 +25,7 @@
   },
   "dependencies": {
     "@vue/test-utils": "^1.0.0-beta.16",
+    "abcjs": "git+https://git@github.com/flxwu/abcjs.git",
```

Comment:
Should point to an actual version. Why not use the official one ?

Reply:
Because we have to modify a css property in the abcjs svg generator to generate overflow:scroll instead of hidden

Reply:
Why not override the css property in StackEdit? (with an `!important` rule I suggest)

Reply:
That's what I initially thought would work as well but abcjs overwrites the parent div's style property entirely